topology/src/main/resources/proto/topology.proto

```diff
@@ -0,0 +1,32 @@
+syntax = 'proto3';
+package topology_protocol;
+
+option java_package = "io.camunda.zeebe.topology.protocol";
+
+message MGossipState {
+  MClusterTopology clusterTopology = 1;
+}
+
+message MClusterTopology {
+  int64 version = 1;
+  map<int32, MMemberState> members = 2;
```

Comment:
:question: Here memberId is an integer. But internally it is a String. So there is a bit of converting between string and integer during serialization and deserialization. Should we just use String here, although we know that all members in the topology has integer ids?